src/modules/feature-modules/admin/components/user-information.component.ts

```diff
@@ -0,0 +1,232 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+import { CoreComponent } from '@app/base';
+import { UserRoleEnum } from '@app/base/enums';
+
+import { UserInfo } from '@modules/shared/dtos/users.dto';
+import { OrganisationsService } from '@modules/shared/services/organisations.service';
+import { AdminUsersService } from '../services/admin-users.service';
+import { UsersValidationRulesService, ValidationRuleEnum } from '../services/users-validation-rules.service';
+
+enum SubmitButton {
+  USER_DETAILS = 'Go to user details',
+  ADD_USER = 'Add user',
+  CONTINUE = 'Continue'
+}
+
+@Component({
+  selector: 'app-admin-pages-user-information',
+  templateUrl: './user-information.component.html',
+})
+export class UserInformationComponent extends CoreComponent implements OnInit {
```

Comment:
A component should NOT extend from CoreComponent.

Only "page" components should do that.